app/posts/manage-training/2023-11-08-teachers-leaving-schools.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Exploring how schools tell us when a teacher has left
+description: We removed an explicit option for induction tutors to add themselves as a mentor, instead allowing them to do it through the normal “add mentor” flow.
+date: 2023-11-08
+tags:
+  - early career teachers
+image:
+  src: /manage-training/teachers-leaving-schools/teachers-leaving-schools.png
+  alt: "Illustration showing the question “Is Jonathan left the school permanently?” with the answers “Yes” or “No, they’ve taken a temporary leave of absence, for example parental leave” as radio options"
+  opengraphImage: true
+---
+
+We did some design and research to better understand how and when schools would tell us when one of their early career teachers leaves their school.
+
+Our research showed that schools did not always understand when they were supposed to do this.
+
+## Current process
+
+Schools using the service see a link on bottom of the page for each early career teacher with the text ‘Tell us if (teacher’s name) is transferring to another school’.
+
+![TODO](current-leaving-link.png)
+
+Clicking this takes them to a confirmation page:
+
+![TODO](current-confirmation-page.png)
+
+If they continue, they are then asked for a date when the teacher left the school. This can be in the past or future:
```

Comment:
```suggestion
If they continue, they are then asked for a date when the teacher has or will leave the school. This can be in the past or future:
```